### PR TITLE
Added preloader page issue-155

### PR DESCRIPTION
**Pull request by ManishaSwain8**
## Related Issue

Added preloader page to the website under GSSOC 2023. 
@Bhushan8177 

Closes: #155 

## Description of Changes

Added preloader page to the website for only 2 seconds

## Screenshots

|      Original       |       Updated        |
| :-----------------: | :------------------: |
| Updated screenshot |  
Screenshot (66)
|

### CODE REVIEW

**Comment by Bhushan8177** [comment]
Hey @ManishaSwain8 , great work but can we go for another cool loader , something that a user will feel is not just a loader !!

**Comment by ManishaSwain8** [comment]
ok got it ! I will come of with something and will open the PR soon  @Bhushan8177

File: client/src/App.css [hunk 1]
```diff
@@ -3,3 +3,41 @@
...
+	justify-content: center !important;
+	align-items: center !important;
+	height: 100vh;
+}
+
+.loader1 {
+	margin-top: -25rem;
+	font-weight: 700;
+	text-align: center;
+	font-size: 25px;
```
Comment (Grab-Bits) [inline client/src/App.css:17]: increase the font size so that it becomes clearly visible and looks  great

**Comment by ManishaSwain8** [comment]
last one push is the updated one, I have added wrong commit msg mistakenly ,it should have been "minor changes issue 155". sorry for inconvenience : /

**Comment by Bhushan8177** [comment]
After a lot number of checks and commits, Finally merging it!
Great work @ManishaSwain8 , Thank you for you contribution ..!

**Comment by Bhushan8177** [comment]
Thank you for Contributing to GrabBit, you can look for other issues you feel can take it to the next level 🚀
Please ⭐ the repository and follow me on Github ...!

You can connect with me on Linkedin - 
https://www.linkedin.com/in/bhushan-kale/

**Comment by ManishaSwain8** [comment]
> Thank you for Contributing to GrabBit, you can look for other issues you feel can take it to the next level 🚀 Please ⭐ the repository and follow me on Github ...!
> 
> You can connect with me on Linkedin - https://www.linkedin.com/in/bhushan-kale/

sure  !

**Comment by ManishaSwain8** [comment]
Hello,@Bhushan8177 there's a little bit problem here,
Screenshot (70)

**Comment by Bhushan8177** [comment]
> Hello,@Bhushan8177 there's a little bit problem here,
> Screenshot (70)
> 

What problem?

**Comment by ManishaSwain8** [comment]
there is the white space below
This is happening due to adding the loader (top:12%)

**Comment by ManishaSwain8** [comment]
Screenshot (72)
 In my opinion you can just do these little changes,

**Comment by Bhushan8177** [comment]
I completely agree to the changes @ManishaSwain8 , I think let's not dive more into css here🤧😂, it is looking great btw !!

**Comment by ManishaSwain8** [comment]
Yup😅

**Comment by ManishaSwain8** [comment]
@Bhushan8177 do add gssoc 2023 tag along with the level of issue.
It will be kind of you.  : )

**Pull request merged**